app/src/main/java/com/github/se/icebreakrr/model/location/LocationService.kt

```diff
@@ -71,34 +121,54 @@ class LocationService(private val fusedLocationProviderClient: FusedLocationProv
       onLocationUpdate: (Location) -> Unit,
       onError: ((String) -> Unit)?
   ): Boolean {
-
     this.onLocationUpdate = onLocationUpdate
     this.onError = onError
-
     try {
-      // Create a location request with high accuracy and specified intervals
       val locationRequest =
           LocationRequest.Builder(Priority.PRIORITY_HIGH_ACCURACY, REQUEST_INTERVAL_MS)
               .setMinUpdateIntervalMillis(MIN_UPDATE_INTERVAL_MS)
               .setWaitForAccurateLocation(true)
               .build()
-
-      // Request location updates, passing the defined callback and main looper
       fusedLocationProviderClient.requestLocationUpdates(
           locationRequest, locationCallback, Looper.getMainLooper())
+      Log.d("LocationService", "Location updates started")
```

Comment:
I don't now if this debug logcat was use for debugging or if it is intended.
If it was only for debugging, you should remove it

Reply:
Same here.